packages/react-devtools-shell/src/devtools.js

```diff
@@ -10,6 +10,10 @@ import {
 import {initialize as initializeFrontend} from 'react-devtools-inline/frontend';
 import {initDevTools} from 'react-devtools-shared/src/devtools';
 
+// This is a pretty gross hack to make the runtime loaded named-hooks-code work.
+// $FlowFixMe
+__webpack_public_path__ = '/dist/'; // eslint-disable-line no-undef
```

Comment:
Yuck 😁 Maybe we can get rid of this once the Webpack 5 upgrade lands (#22267)

Reply:
Would everyone who uses the inline package now have to do this now?